### PR TITLE
Feature/ET-933 Add cross device support for useCustomizedApiRequests

### PR DESCRIPTION
**Pull request by eraxyz**
# Problem

The new useCustomizedApiRequests feature for network decoupling does not work on the cross-device flow due to the option not being validated and triggering the appropriate action. 

# Solution

Added code to trigger the setDecoupleFromAPI action if the useCustomizedApiRequests options is present and valid in the CrossDeviceMobileRouter

## Checklist

_put `n/a` if item is not relevant to PR changes_

- [x] Has the CHANGELOG been updated?
- [x] Has the README been updated?
- [ ] Has the CONTRIBUTING doc been updated?
- [ ] Has the RELEASE_GUIDELINES been updated?
- [ ] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [ ] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [ ] Have any new automated tests been implemented or the existing ones changed?
- [ ] Have any new manual tests been written down or the existing ones changed?
- [ ] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1372-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1372-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1372-pr-onfido-sdk-ui-onfido.surge.sh

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 - Internal: Added user consent screen
 - Public: Added callbacks that are triggered on user media submission if the feature is enabled. Note - This is a premium enterprise feature.
 - Internal: Introduce `SdkOptionsProvider` component and `useSdkOptions()` hook for SDK options' single source of truth.
+- Public: Add cross-device support for useCustomizedApiRequests callbacks via customer hosting of SDK.
```
Comment (kopijunkie) [inline CHANGELOG.md:17]: Think it needs to be made clear that this relates to the "premium enterprise feature" in line 15.

File: README.md [hunk 2]
```diff
@@ -806,6 +795,145 @@ VIDEO_FACIAL_CAPTURE_STEP_2 - User reached the 2nd challenge during "liveness vi
...
+ 		detect_document: string,
+		detect_cutoff: string,
+    detect_glare: string,
+    detect_blur: string,
+  },
+  sdk_source: string,
+  sdk_version: string,
+  sdk_metadata: {
+    captureMethod: string,
+    imageResizeInfo: string | null,
```
Comment (kopijunkie) [inline README.md:846]: `imageResizeInfo` is actually an object if provided, e.g. 
```
"imageResizeInfo": {
    "resizedFrom":
    {
        "width": 7016,
        "height": 4951,
        "fileSize": 10544735
    },
    "resizedTo": {
        "width": 1440,
        "height": 1016.1687571265678,
        "fileSize": 426058
    }
}
```

**Review by kopijunkie — CHANGES_REQUESTED** [review]

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1372-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1372-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1372-pr-onfido-sdk-ui-onfido.surge.sh

File: README.md [hunk 3]
```diff
@@ -806,6 +795,598 @@ VIDEO_FACIAL_CAPTURE_STEP_2 - User reached the 2nd challenge during "liveness vi
...
+onSubmitDocument: (data) => {
+  // Send request to Onfido API /documents via your backend proxy
+  .then(onfidoSuccessResponse =>
+    Promise.resolve({ onfidoSuccessResponse }))
+  .catch(onfidoError => Promise.reject(onfidoError))
+}
+
+```
+
+Below is a sample openAPI yml file you could use as an example to start your own proxy.
```
Comment (kopijunkie) [inline README.md:942]:
```suggestion
Below is a sample openAPI YAML file you could use as an example to start your own proxy.
```

**Review by kopijunkie — APPROVED** [review]
LGTM. I think having the Enterprise Features section in the README is good as a single point of reference for integrators, but this doc is growing to be a bit of a beast IMO. It'll become even bigger once the UI customisation documentation's merged in. 😅

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1372-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1372-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1372-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1372-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1372-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1372-pr-onfido-sdk-ui-onfido.surge.sh

File: CHANGELOG.md (hunk 1 above)
Comment (eraxyz) [reply CHANGELOG.md]: Done, thanks

File: README.md (hunk 2 above)
Comment (eraxyz) [reply README.md]: Thanks for pointing this out. I made the change but it's been decided now that we should just have sdk_metadata as `object` so I've changed it to that now.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1372-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1372-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1372-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1372-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1372-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1372-pr-onfido-sdk-ui-onfido.surge.sh

**Review by ethan605 — APPROVED** [review]
LGTM!

**Pull request merged**